api/v1alpha1/ratelimit_types.go

```diff
@@ -138,6 +138,12 @@ type SourceMatch struct {
 	// +kubebuilder:validation:MinLength=1
 	// +kubebuilder:validation:MaxLength=256
 	Value string `json:"value"`
+
+	// InvertMatch specifies whether the value match result will be inverted.
+	//
+	// +optional
+	// +kubebuilder:default=false
+	InvertMatch bool `json:"invertMatch,omitempty"`
```

Comment:
Prefer just `Invert`

Reply:
Also should be a pointer since it's an optional type

Reply:
@shawnh2 I am not sure what `optional` means here, want this param to be optional for user to set but default interpretation should be `false`, implementation shouldn't have to check a for nil pointer. Let me know what you think!